core/src/main/scala/org/apache/spark/deploy/worker/Worker.scala

```diff
@@ -121,7 +121,7 @@ private[spark] class Worker(
   val shuffleService = new StandaloneWorkerShuffleService(conf, securityMgr)
 
   val publicAddress = {
```

Comment:
This is probably fine as-is, though, since this `publicAddress` is used to construct the worker web UI links in `WorkerInfo.scala`.